### DISCUSSION TITLE
AWS Lambda: @prisma/client did not initialize yet. Please run \"prisma generate\" and try to import it again.

### DISCUSSION THREAD

**Discussion by coulonxyz** [prisma/prisma#24028] [Q&A]
### Bug description

I'm trying to configure Prisma on our backend to replace typeorm. It works fine locally, but when i deploy it, i have the following error: @prisma/client did not initialize yet. Please run \"prisma generate\" and try to import it again.
I've listed directory content, and i can see that `libquery_engine-rhel-openssl-1.0.x.so.node` and `schema.prisma` are present in `./node_modules/.prisma/client`. I'm lost here, and we are about to cancel the migration to Prisma.

Anyone has an idea of what could cause this error ?

### How to reproduce

1. push repository to github to trigger deployment via circle ci
2. install: 
``` shell
npm ci
npx prisma generate
```
3. deploy:
```shell
npx serverless deploy --stage develoment
```

4. init Prisma client with the following config:

```typescript
new PrismaClient({
        log: ['info', 'warn', 'error'],
        datasources: {
          db: {
            url: postgres://login:password@foo.bar.us-east-1.rds.amazonaws.com:5432/mydatabase_development?timezone=UTC,
          },
        },
      })

```

and get this:

```
    "error": {
            "stack": "Error: @prisma/client did not initialize yet. Please run \"prisma generate\" and try to import it again.\nIn case this error is unexpected for you, please report it in https://pris.ly/prisma-prisma-bug-report\n    at new PrismaClient (/var/task/node_modules/.prisma/client/default.js:43:11)\n    at Function.getInstance (webpack://ramdam-front-exposition-api/src/database/prisma/client.ts:11:34)\n    at getPrismaClient (webpack://ramdam-front-exposition-api/src/database/prisma/client.ts:24:54)\n    at <anonymous> (webpack://ramdam-front-exposition-api/src/api/users/controllers/get-me.ts:35:35)\n    at Generator.next (<anonymous>)\n    at fulfilled (webpack://ramdam-front-exposition-api/src/api/users/controllers/delete-me.ts:20:2)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
```

### Expected behavior

prisma to be intialized

### Prisma information

```ts
new PrismaClient({
        log: ['info', 'warn', 'error'],
        datasources: {
          db: {
            url: postgres://login:password@foo.bar.us-east-1.rds.amazonaws.com:5432/mydatabase_development?timezone=UTC,
          },
        },
      })
```

### Environment & setup

- OS: AWS lambda
- Database: Postgres
- Node.js version: node18

```
INIT_START Runtime Version: nodejs:18.v28	Runtime Version ARN: arn:aws:lambda:us-east-1::runtime:b475b23763329123d9e6f79f51886d0e1054f727f5b90ec945fcb2a3ec09afdd
```

### Prisma Version

```
    "@prisma/client": "^5.13.0",
    "prisma": "^5.13.0",
```

**Comment by ludralph** [comment]
Hi @coulonxyz 👋 

Thank you for raising this question.

Are you following this [guide](https://www.prisma.io/docs/orm/prisma-client/deployment/serverless/deploy-to-aws-lambda) to deploy to AWS Lambda? Did you include the `prisma generate` script in the serverless.yml file?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
